### PR TITLE
Create climate.mill.markdown

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17313
## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @Danielhiversen 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/climate.mill.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+sharing: true
+footer: true
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Cloud Polling"
+---
+
+Integrates Mill heater into Home Assistant.
+
+The component requires the heater is sat as an independent device in the Mill app:
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown]: ... requires that the heater is set as an independent device...

File: source/_components/climate.mill.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```yaml
+climate:
+ - platform: mill
+   username: email@gmail.com
+   password:  pswd
+```
+
+{% configuration %}
+username:
+  description: Your registred Mill email.
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown]: registered

File: source/_components/climate.mill.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+description: "Instructions on how to integrate Mill heater into Home Assistant."
+date: 2018-10-10 15:00 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Cloud Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown:14]: No logo?

File: source/_components/climate.mill.markdown [hunk 4]
```diff
@@ -14,7 +14,7 @@ ha_iot_class: "Cloud Polling"
 
 Integrates Mill heater into Home Assistant.
 
-The component requires the heater is sat as an independent device in the Mill app: 
+The component requires that the heater is sat as an independent device in the Mill app:
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown]: sat -> set

**Review by MartinHjelmare — DISMISSED** [review]
Nice!

File: source/_components/climate.mill.markdown [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+The component requires that the heater is set as an independent device in the Mill app: 
+https://millheat.zendesk.com/hc/en-us/articles/115001123491-What-is-an-Independent-device-
+
+
+To enable this platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+climate:
+ - platform: mill
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown]: Maybe add a space before the dash, to make the indentation 2 space?

File: source/_components/climate.mill.markdown [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+logo: mill.png
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Cloud Polling"
+---
+
+Integrates Mill heater into Home Assistant.
+
+The component requires that the heater is set as an independent device in the Mill app: 
+https://millheat.zendesk.com/hc/en-us/articles/115001123491-What-is-an-Independent-device-
```
Comment (MartinHjelmare) [inline source/_components/climate.mill.markdown]: This url isn't a link. Do we want it to be a link?

**Review by MartinHjelmare — COMMENTED** [review]
Just looked at the preview and found some more things.

**Comment by Danielhiversen** [comment]
Where do you see the preview?

**Comment by MartinHjelmare** [comment]
When the build is done at netlify they serve us a preview of the PR of the whole site. Just click the build link. It's awesome!

**Review by MartinHjelmare — DISMISSED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**